net/basic_socket.h

```diff
@@ -172,6 +174,66 @@ inline int get_peer_name(int fd, char* path, size_t count) {
     return do_get_name(fd, &::getpeername, path, count);
 }
 
+// sockaddr_storage is the container for any socket address type.
+// It can be used to convert Endpoint to sockaddr or socklen_t that libc's network API requires.
+struct sockaddr_storage {
```

Comment:
`sockaddr_storage` moved to `basic_socket.h`, because for now there are no other methods to get `sockaddr*` and `socklen_t`.